03-task-manager/starter/routes/task.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+const {getAllTasks,addTask,getTask,deleteTask,updateTask} = require('../controllers/task');
+
+router.route('/').get(getAllTasks).post(addTask)
+
+router.route('/:id').delete(deleteTask).patch(updateTask).get(getTask)
+
+
```

Comment:
It is not good practice to have all these extra spaces unless you have a reason.